src/main/java/com/epam/test_generator/dto/SuitCaseUpdateDTO.java

```diff
@@ -0,0 +1,70 @@
+package com.epam.test_generator.dto;
+
+import com.epam.test_generator.entities.Status;
+import java.util.List;
+
+/**
+ *  This class is needed as the return value in updateCase and updateSuit methods.
+ *  It contains two field. The first can be {@link SuitDTO} or {@link CaseDTO} and the second is
+ *  {@link List<Long>} (in fact id of {@link StepDTO} with FAILED {@link Status} which belong this
+ *  suit or case)
+ */
+public class SuitCaseUpdateDTO<T> {
```

Comment:
Better use separate DTO's for suits and cases

Reply:
Fixed